functions/zoom-downloader.py

```diff
@@ -27,6 +27,14 @@
 CLASS_SCHEDULE_TABLE = env("CLASS_SCHEDULE_TABLE")
 LOCAL_TIME_ZONE = env("LOCAL_TIME_ZONE")
 
+# Recordings that happen within BUFFER_MINUTES a courses schedule
+# start time will be captured
+BUFFER_MINUTES = 30
+# Ignore recordings that are less than MIN_DURATION (in minutes)
+MINIMUM_DURATION = 2
```

Comment:
I trust that this will catch the majority of "ooops" videos? I mean, a 3 minute recording also seems like probably a mistake too.

Reply:
True. We could probably increase it, maybe to 4. But looking at cloudwatch dashboard, most recordings <5 minutes are <2 minutes long.